### DISCUSSION TITLE
/posts/string-piece-uaf

### DISCUSSION THREAD

**Discussion by giscus** [VOID001/VOID001.github.io#3] [Announcements]
# /posts/string-piece-uaf

TL;DR

https://void-shana.moe//posts/string-piece-uaf

**Comment by Endle** [comment]
是不是C++写到尽头，加上了各种lifetime check就成了rust（笑）

**Reply by VOID001** [reply]
哈哈，我觉得会的，而且他们还做不到编译期间检查，太难受了
